ehsm_kms_service/enroll_app/enroll_msg.cpp

```diff
@@ -846,22 +843,49 @@ enroll_status_t ra_proc_msg3_get_att_result_msg(RetJsonObj retJsonObj_msg3, std:
     return ret;
 }
 
-enroll_status_t verify_apikey_result_msg(RetJsonObj retJsonObj_apikey_result_msg)
+enroll_status_t ra_proc_apikey_result_msg_get_apikey(RetJsonObj retJsonObj_apikey_result_msg, uint8_t *apikey)
 {
+    // check nonce
     sgx_quote_nonce_t nonce;
+    enroll_status_t enroll_ret = ENL_OK;
+    uint8_t *iv = nullptr;
+    uint8_t *mac = nullptr;
+    uint32_t ret = 0;
+    uint8_t cipherapikey[EH_API_KEY_SIZE + SAMPLE_SP_IV_SIZE + SAMPLE_AESGCM_MAC_SIZE] = {0};
+
```

Comment:
check the input parameters

Reply:
you can add a open and do it in next patch

Reply:
add check code